src/test/java/org/opensearch/performanceanalyzer/ConfigOverridesIT.java

```diff
@@ -49,12 +49,24 @@
     private static final String CONFIG_OVERRIDES_ENDPOINT =
             RestConfig.PA_BASE_URI + "/override/cluster/config";
 
+    private static final String LEGACY_CONFIG_OVERRIDES_ENDPOINT =
```

Comment:
Looks suspiciously similar to `CONFIG_OVERRIDES_ENDPOINT` :) Maybe the values _should_ be different?

If these are OpenDistro legacy, change to `LEGACY_OPENDISTRO_CONFIG_OVERRIDES_ENDPOINT`.

Reply:
Done.